src/controllers/midi/midicontroller.cpp

```diff
@@ -537,3 +547,41 @@ void MidiController::processInputMapping(const MidiInputMapping& mapping,
     qCWarning(m_logBase) << "MidiController: No script function specified for"
                          << MidiUtils::formatSysexMessage(getName(), data, timestamp);
 }
+
+void MidiController::makeInputHandler(const QJSValue& control, const QJSValue& scriptCode) {
+    if (!scriptCode.isCallable()) {
+        QString sErrorMessage(
+                "Invalid timer callback provided to midi.makeInputHandler. "
+                "Please pass a function and make sure that your code contains no syntax errors.");
```

Comment:
make `const` and put into an anonymous namespace at the top of the file instead.